control-plane-operator/controllers/hostedcontrolplane/kcm/config.go

```diff
@@ -58,13 +62,20 @@ func generateConfig(serviceServingCA *corev1.ConfigMap) (string, error) {
 	return string(b), nil
 }
 
-func ReconcileRecyclerConfig(config *corev1.ConfigMap, ownerRef config.OwnerRef) error {
+func ReconcileRecyclerConfig(config *corev1.ConfigMap, ownerRef config.OwnerRef, releaseImageProvider *imageprovider.ReleaseImageProvider) error {
+	var result strings.Builder
+
 	ownerRef.ApplyTo(config)
 	if config.Data == nil {
 		config.Data = map[string]string{}
 	}
+
+	data := map[string]string{
+		"rhtoolsImageName": releaseImageProvider.GetImage("tools"),
```

Comment:
should we add a check to make sure that the image exists?

Reply:
I don't think that's necessary meanwhile it's part of the base in the ocp payload.